lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -783,6 +883,16 @@ void HdVP2Material::_UpdateShaderInstance(const HdMaterialNetwork& mat)
                         status = _surfaceShader->setParameter(paramName,
                             info._isColorSpaceSRGB);
                     }
+                    if (status) {
+                        paramName = nodeName + "stScale";
+                        const float* val = &(info._stScale[0]);
+                        status = _surfaceShader->setParameter(paramName, val);
+                    }
+                    if (status) {
+                        paramName = nodeName + "stOffset";
+                        const float* val = &(info._stOffset[0]);
+                        status = _surfaceShader->setParameter(paramName, val);
+                    }
```

Comment:
If you use GfVec2f, the code can be simplified to:

`status = _surfaceShader->setParameter(paramName, info._stScale.data());`